install/boards/bcm_27xx.sh

```diff
@@ -6,16 +6,33 @@ VERSION="${VERSION:-master}"
 GITHUB_REPOSITORY=${GITHUB_REPOSITORY:-bluerobotics/blueos-docker}
 REMOTE="${REMOTE:-https://raw.githubusercontent.com/${GITHUB_REPOSITORY}}"
 ROOT="$REMOTE/$VERSION"
-CMDLINE_FILE=/boot/cmdline.txt
 alias curl="curl --retry 6 --max-time 15 --retry-all-errors"
 
+# Determine OS
+OS=$(lsb_release -is)
+
+if [ "$OS" = "Raspbian" ]; then
+    echo "Running on Raspbian."
+    CMDLINE_FILE="/boot/cmdline.txt"
+    CONFIG_FILE="/boot/config.txt"
+    OVERLAYS_DIR="/boot/overlays"
+elif [ "$OS" = "Ubuntu" ]; then
+    echo "Running on Ubuntu."
+    CMDLINE_FILE="/boot/firmware/cmdline.txt"
+    CONFIG_FILE="/boot/firmware/config.txt"
+    OVERLAYS_DIR="/boot/firmware/overlays"
+else
+    echo "Unknown OS"
+    exit 1
```

Comment:
We should not abort, we could try to check if the files exist and try to select it automatically or provide fault ones.
But in both cases, warning the user that the system is not supported.